packages/mdl-fab/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "mdl-fab",
+  "version": "1.0.0",
+  "license": "Apache 2.0"
```

Comment:
You need to specify `Apache-2.0` as that's the correct SPDX license [identifier](https://spdx.org/licenses/Apache-2.0.html).